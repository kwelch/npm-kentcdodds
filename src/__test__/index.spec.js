import kwelch from '../index';

describe('kwelch', () => {
  it('should return consistent object', () => {
    expect(kwelch).toMatchSnapshot();
  });
});
