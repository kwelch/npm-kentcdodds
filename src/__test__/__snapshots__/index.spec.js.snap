// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kwelch should return consistent object 1`] = `
Object {
  "birthday": "1990-06-26",
  "email": "kyle@krwelch.com",
  "github": "https://github.com/kwelch",
  "name": Object {
    "first": "Kyle",
    "last": "Welch",
    "middle": "Ryan",
  },
  "twitter": "https://twitter.com/kylewelch",
  "website": "http://krwelch.com/",
}
`;
